kubelet/datadog_checks/kubelet/kubelet.py

```diff
@@ -416,7 +416,10 @@ def _report_container_spec_metrics(self, pod_list, instance_tags):
                 if self.pod_list_utils.is_excluded(cid, pod_uid):
                     continue
 
-                tags = tagger.tag('%s' % cid, tagger.HIGH) + instance_tags
+                tags = tagger.tag('%s' % cid, tagger.HIGH)
```

Comment:
same here:

```
tags = tagger.tag('%s' % cid, tagger.HIGH) or [] + instance_tags
```

Reply:
WDYT about splitting in 2 lines for readability on this one?